test/test-pools.json

```diff
@@ -55,196 +55,224 @@
   ],
   "pools": [
     {
-      "name": "S:100:APT:tAPT",
```

Comment:
do we not have coin type in pool object?

Reply:
yes we have. The `poolType` here is parsed from pool object